src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -179,4 +187,26 @@ function nextPageButtonHandler({
   }
 }
 
+function processRedirect(appState: any): void {
+  // All logic for re-directing/configuring page based on application state, permissions, roles, etc. should go here.
+  const {
+    stage,
+    status,
+    outcome,
+    serialNumber,
+    sectionCode,
+    page,
+    templateSections,
+    push,
+  } = appState
+  if (status !== 'DRAFT') {
+    push(`/applications/${serialNumber}/summary`)
```

Comment:
should this be 'replace' ?

Reply:
Yeah, good call.

Reply:
looks good, I think there might be some artefact `push` being passed along and imported